.travis.yml

```diff
@@ -1,16 +1,10 @@
 language: php
-dist: precise
-sudo: false
+dist: bionic
 
 php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7.0
-  - 7.1
-  - 7.2
   - 7.3
+  - 7.4
+  - 8.0
```

Comment:
I'd like to keep all version if possible, unless there is a reason to remove an old one (this PR doesn't change anything in the code, so that removing old version is not needed).

Reply:
well, fine by me. but keep https://www.php.net/supported-versions.php

the only problem I see is that you can't have such a wide version range with Travis. See https://docs.travis-ci.com/user/languages/php/#php-versions. Either you have an old distro with old php version or a newer one with new versions. There is none that covers 5.3-8.0

Reply:
> Either you have an old distro with old php version or a newer one with new versions. There is none that covers 5.3-8.0

correct. we can define the "dist" per version thought (we should actually)
see eg https://github.com/symfony/symfony/blob/3.4/.travis.yml#L27

Reply:
thanks a lot, didn't know this was possible